Makefile

```diff
@@ -158,11 +158,19 @@ endif
 
 OBJDUMP_FLAGS := -d -r -z -Mreg-names=32
 
+# rom compression flags
+COMPFLAGS := --threads $(N_THREADS)
+ifneq ($(NON_MATCHING),1)
+  COMPFLAGS += --matching
+endif
+
 #### Files ####
 
 # ROM image
 ROM := oot-$(VERSION).z64
-ELF := $(ROM:.z64=.elf)
+ROMC := oot-$(VERSION).z64
+ROM := oot_uncompressed-$(VERSION).z64
```

Comment:
I find the ROM naming a bit inconsistent. Right now there's

```
baserom.gc-eu-mq-dbg.z64
baserom_uncompressed.gc-eu-mq-dbg.z64
oot-gc-eu-mq-dbg.z64
oot_uncompressed-gc-eu-mq-dbg.z64
```

There's a mix of underscores and dashes, and the version is separated  by a period in `baserom.` but by a dash in `oot-`.

One option is to go all in on periods, e.g. something like

```
baserom.gc-eu-mq-dbg.z64
baserom.gc-eu-mq-dbg.uncompressed.z64
oot.gc-eu-mq-dbg.z64
oot.gc-eu-mq-dbg.uncompressed.z64
```

although I'm not totally sold on that

Reply:
personally not a fan of too many dots, my brain automatically associates things after dots as file extensions. but I dont have an alternative solution to offer at this time

Reply:
A dotless way could be something like

```
baserom-gc-eu-mq-dbg.z64
baserom-gc-eu-mq-dbg-uncompressed.z64
oot-gc-eu-mq-dbg.z64
oot-gc-eu-mq-dbg-uncompressed.z64
```

Reply:
I would like baseroms to go in a subfolder tbh, since there are a lot of them and even variants like "pal mq debug with overdump", "pal mq debug stripped", "1.0 compressed", "1.0 uncompressed" ...

We can also move the md5 sums to that subfolder. And relevant scripts

And the current baserom/ folder to `baseroms/segments/{version}/` ? idk

Reply:
I like just dashes personally

Reply:
so with subfolders, youd rather just name them "baserom.z64" and let the folder handle the versioning?

Reply:
![image](https://github.com/zeldaret/oot/assets/22045047/dfeb258f-53c7-4434-938d-0eff20e29c5e)
im fine with this approach mentioned on discord

Reply:
Sorry, what was the resolution for the output rom names?

Reply:
None so far, still being discussed in https://github.com/zeldaret/oot/pull/1614#discussion_r1456907385

Reply:
Oh I didn't realize you were trying to combine the discussions, sorry

Reply:
They are the same conversation right? Just wanna make sure I'm not missing something

Reply:
In my mind they were slightly different (ROM naming vs semantics of the build system) so I was a bit confused why you resolved this, but combining them is fine by me. I should have DMed you instead